endstate_rew/system.py

```diff
@@ -6,6 +6,9 @@
 from openmm.app import Simulation
 from openmmml import MLPotential
 from tqdm import tqdm
+from openmm.app import CharmmPsfFile, CharmmCrdFile, CharmmParameterSet
+from openmm.app import NoCutoff
+from openmm.unit import unit
```

Comment:
I think the import has to be `from openmm import unit` instead of `from openmm.unit import unit`.